### PR TITLE
Specimen comparison improvements

### PR DESCRIPTION
**Pull request by ORybak5**
Adding more field to ignore to decrease the amount of false positive results.

### CODE REVIEW

**Comment by adrianclay** [comment]
I find it hard to review this change, as there's no explanation as to why the fields have been ignored.  I think it would be useful to add that context, so that I can agree/disagree!

File: raw-compare.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const observationFieldsToIgnore = [];
 const practitionerFieldsToIgnore = ["resource.meta.versionId", "resource.gender"];
 const procedureRequestFieldsToIgnore = [];
 const diagnosticReportFieldsToIgnore = [];
-const specimenFieldsToIgnore = [];
+const specimenFieldsToIgnore = ["resource.status", "resource.receivedTime", "resource.identifier"];
```
Comment (ORybak5) [inline raw-compare.js:26]: These fields will be ignored in comparison:
- status field is not copied
- received time is difficult to include into comparison as it is prone to errors due to timestamp. collectedDateTime field is included istead.
- identifier is also not included to make it consistent with other types where this field is not included.

File: raw-compare.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ const listMatchFields = ["code.coding.[0].code", "date", "title", "id"];
 const locationMatchFields = ["name"];
 const conditionMatchFields = ["onsetDateTime", "assertedDate", "code.coding.[0].code"];
 const procedureRequestMatchFields = ["id"];
-const specimenMatchFields = ["accessionIdentifier.value", "type.text"];
+const specimenMatchFields = ["accessionIdentifier.value", "type.text", "collection.collectedDateTime"];
```
Comment (ORybak5) [inline raw-compare.js:106]: Additing collectedDateTime field to be included in comparison

File: raw-compare.js [hunk 3]
```diff
@@ -557,7 +557,7 @@ function setupCompare(emisEntries, psEntries, resourceName, resourceTypeIndex, a
 
 			// if we did not find a match...
 			outputReport = outputReport + "EMIS " + resourceName + " Entry " + (emisEntryCount + 1) + "\n";
-			outputReport = outputReport + "MISSING Entry: EMIS entry with ID " + emisEntryId + " has not been found in the PS enties \n"
+			outputReport = outputReport + "MISSING Entry: EMIS entry with ID " + emisEntryId + " has not been found in the PS entries \n"
```
Comment (ORybak5) [inline raw-compare.js:560]: Fixing a small string typo

**Comment by ORybak5** [comment]
> 

@adrianclay I've included some additional comments as requested, hope you find it helpful.

**Review by benhession — APPROVED** [review]

**Pull request merged**